test/libsolidity/semanticTests/libraries/library_call_in_homestead.sol

```diff
@@ -0,0 +1,13 @@
+library Lib { function m() public returns (address) { return msg.sender; } }
+contract Test {
+    address public sender;
+    function f() public {
+        sender = Lib.m();
+    }
+}
+// ====
+// compileViaYul: also
+// ----
+// library: Lib
+// f() ->
+// sender() -> 0x1212121212121212121212121212120000000012
```

Comment:
@chriseth Do we have a helper for library addresses or do we always just hard-code them?

Reply:
I think this is the hardcoded msg.sender - isn't it?

Reply:
Is this okay ? Can we add this to our PR , or is there some change needed ?